lib/sandbox/sandbox_unsupported.go

```diff
@@ -0,0 +1,50 @@
+// +build !linux
+
+package sandbox
+
+import (
+	"fmt"
+	"os"
+)
+
+func isNSorErr(nspath string) error {
+	return nil // TODO(vbatts) ... really not sure ...
```

Comment:
I recommended raising a not-implemented error here too.

Reply:
k